fabric/src/main/java/net/minescript/common/Minescript.java

```diff
@@ -3981,6 +4010,116 @@ private static Optional<String> handleScriptFunction(
+            if (itemStack.isEmpty()) {
+              continue;
+            }
+            resultString.append(itemStackToJsonString(itemStack, OptionalInt.of(items[i].id), false));
+            if (i < items.length - 1) {
+              resultString.append(",");
+            }
+          }
+          resultString.append("]");
+          return Optional.of(resultString.toString());
+        } else {
+          return Optional.empty();
+        }
+      }
+
+      case "container_click_slot": // Click on slot in container
+      {
+        if (args.size() != 1) {
+          logUserError("Error: `{}` expected 1 param (slot id) but got: {}", functionName, argsString);
+          return Optional.empty();
+        }
+        Screen screen = minecraft.currentScreen;
+        int slot = getStrictIntValue(args.get(0)).getAsInt();
+        if (screen == null) {
+          return Optional.of("false");
+        }
+        if (screen instanceof HandledScreen<?>) {
+          HandledScreen handledScreen = (HandledScreen) screen;
+          ScreenHandler screenHandler = handledScreen.getScreenHandler();
+          if (screenHandler.slots.size() <= slot) {
+            logUserError("Error: `{}`: slot {} does not exist", functionName, slot);
+            return Optional.of("false");
+          }
+          //screenHandler.onSlotClick(slot, 0, SlotActionType.PICKUP, minecraft.player);
+          //screenHandler.sendContentUpdates();
+          PacketByteBuf buf = new PacketByteBuf(Unpooled.buffer());
+          buf.writeByte(screenHandler.syncId);
+          buf.writeShort(slot);
+          buf.writeByte(0); // Button (0 for left click)
+          buf.writeShort(0); // Action type (0 for click)
+          buf.writeShort(0); // Mode (0 for normal)
+          buf.writeItemStack(screenHandler.getSlot(slot).getStack()); // Item stack
+          minecraft.getNetworkHandler().sendPacket(new ClickSlotC2SPacket(buf));
+          screenHandler.sendContentUpdates();
+          return Optional.of("true");
+        } else {
+          return Optional.of("false");
+        }
+      }
+
+      case "player_look_at_position": // Look at x, y, z
+      {
+        if (args.size() != 3) {
+          logUserError("Error: `{}` expected 3 params (x, y, z) but got: {}", functionName, argsString);
+          return Optional.empty();
+        }
+        OptionalDouble value1 = getStrictDoubleValue(args.get(0));
+        if (!value1.isPresent()) {
+          logUserError("Error: `{}` expected first param to be a number but got: {}", functionName, argsString);
+          return Optional.empty();
+        }
+        OptionalDouble value2 = getStrictDoubleValue(args.get(1));
+        if (!value2.isPresent()) {
+          logUserError("Error: `{}` expected second param to be a number but got: {}", functionName, argsString);
+          return Optional.empty();
+        }
+        OptionalDouble value3 = getStrictDoubleValue(args.get(2));
+        if (!value3.isPresent()) {
```

Comment:
Rather than handling individual errors for value1 through value3, please check them with one compound condition similar to `"player_set_orientation"`, but throw `IllegalArgumentException` instead.